### PR TITLE
DM-6640: IsrTask is not a valid CmdLineTask

### PR DESCRIPTION
**Pull request by czwa**
This ticket adds a stand-alone runIsr.py script, as well as ensuring that runIsr.py and processCcd.py read the same ISR configuration files.

### CODE REVIEW

File: config/hsc/isr.py [hunk 1]
```diff
@@ -13,6 +14,9 @@
 config.doDark = True # Required especially around CCD 33
 config.doFringe = True
 config.fringe.filters = ['y', 'N921', 'N926', 'N973', 'N1010']
+
+# Do not use NO_DATA pixels for fringe subtraction.
+config.fringe.stats.badMaskPlanes = ['SAT', 'NO_DATA']
```
Comment (PaulPrice) [inline config/hsc/isr.py]: This looks like an unrelated functional change that should be in a separate commit and justified.

File: config/hsc/isrWrap.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+"""
+HSC-specific overrides for IsrWrapperTask
+"""
+import os.path
+
+from lsst.utils import getPackageDir
+from lsst.obs.subaru.isr import SubaruIsrTask
+
+ObsConfigDir = os.path.join(getPackageDir("obs_subaru"), "config", "hsc")
```
Comment (PaulPrice) [inline config/hsc/isrWrap.py]: Leading Uppercase is reserved for class names.

File: config/hsc/processCcd.py [hunk 3]
```diff
@@ -7,17 +7,19 @@
...
+
+config.calibrate.photoCal.colorterms.load(os.path.join(ObsConfigDir, 'colorterms.py'))
 config.charImage.measurePsf.starSelector["objectSize"].widthMin = 0.9
 config.charImage.measurePsf.starSelector["objectSize"].fluxMin = 4000
 for refObjLoader in (config.calibrate.astromRefObjLoader,
                      config.calibrate.photoRefObjLoader,
                      config.charImage.refObjLoader,
                      ):
-    refObjLoader.load(os.path.join(hscConfigDir, "filterMap.py"))
+    refObjLoader.load(os.path.join(ObsConfigDir, "filterMap.py"))
```
Comment (PaulPrice) [inline config/hsc/processCcd.py]: The changes in this file seem unnecessary.

File: config/processCcd.py [hunk 4]
```diff
@@ -67,18 +68,18 @@
...
 if "ext_shapeHSM_HsmShapeRegauss" in config.charImage.measurement.plugins:
     # no deblending has been done
     config.charImage.measurement.plugins["ext_shapeHSM_HsmShapeRegauss"].deblendNChild = ""
 
-config.calibrate.measurement.load(os.path.join(configDir, "apertures.py"))
-config.calibrate.measurement.load(os.path.join(configDir, "kron.py"))
-config.calibrate.measurement.load(os.path.join(configDir, "hsm.py"))
+config.calibrate.measurement.load(os.path.join(ObsConfigDir, "apertures.py"))
+config.calibrate.measurement.load(os.path.join(ObsConfigDir, "kron.py"))
+config.calibrate.measurement.load(os.path.join(ObsConfigDir, "hsm.py"))
```
Comment (PaulPrice) [inline config/processCcd.py]: Again, changes in this file are unnecessary.

File: config/suprimecam-mit/isrWrap.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+"""
+Suprimecam-specific overrides for IsrWrapperTask
+"""
+import os.path
+
+from lsst.utils import getPackageDir
+from lsst.obs.subaru.isr import SuprimeCamIsrTask
+
+ObsConfigDir = os.path.join(getPackageDir("obs_subaru"), "config", "suprimecam")
```
Comment (PaulPrice) [inline config/suprimecam-mit/isrWrap.py]: `"suprimecam-mit"`

Comment (czwa) [reply config/suprimecam-mit/isrWrap.py]: Thanks for catching this.

File: config/hsc/isr.py (hunk 1 above)
Comment (czwa) [reply config/hsc/isr.py]: This was pulled from `config/hsc/processCcd.py`, which listed these values for `config.isr.fringe.stats.badMaskPlanes`.  Rereading the version on master, I'm now not convinced that the `+= ["SUSPECT"]` line isn't the change in functionality.  I will try to find a test.
